app/config/logging-cfg-prod.yml

```diff
@@ -4,10 +4,14 @@ disable_existing_loggers: False # this allow to get logger at module level
 root:
   handlers:
     - console
-  level: INFO
+  level: WARNING
   propagate: True
 
 loggers:
+  stac_api:
+    level: DEBUG
+  middleware:
+    level: DEBUG
   django:
     level: INFO
```

Comment:
You might want to remove these lines, otherwise django will use INFO level instead of WARNING.

Reply:
Yeah, thought about that. Not sure if these messages will be useful or not. I don't think that we'll use them... but we can leave it for the moment and remove once we see that things are stable and we never use them.